ResearchKit/Common/ORKScaleSliderView.m

```diff
@@ -128,79 +159,68 @@ - (void)updateConstraints {
-                                                                      relatedBy:NSLayoutRelationEqual
-                                                                         toItem:self.slider
-                                                                      attribute:NSLayoutAttributeCenterX
-                                                                     multiplier:1.0
-                                                                       constant:-kBigMargin];
-        [self addConstraint:constraint];
-        constraint = [NSLayoutConstraint constraintWithItem:self.leftRangeLabel
-                                                  attribute:NSLayoutAttributeRight
-                                                  relatedBy:NSLayoutRelationEqual
-                                                     toItem:self.slider
-                                                  attribute:NSLayoutAttributeCenterX
-                                                 multiplier:1.0
-                                                   constant:-kBigMargin];
-        [self addConstraint:constraint];
-    }
-    else
-    {
-        // Horizontal slider constraints
-        const CGFloat kMargin = 17.0;
-        [self addConstraints:[NSLayoutConstraint constraintsWithVisualFormat:@"V:|[_valueLabel(==40)]-[_slider]"
-                                                                     options:NSLayoutFormatAlignAllCenterX
-                                                                     metrics:nil
-                                                                       views:views]];
+        [self addConstraint:[NSLayoutConstraint constraintWithItem:self.leftRangeLabel
+                                                         attribute:NSLayoutAttributeRight
+                                                         relatedBy:NSLayoutRelationEqual
+                                                            toItem:self.slider
+                                                         attribute:NSLayoutAttributeCenterX
+                                                        multiplier:1.0
+                                                          constant:-kSideLabelMargin]];
         
-        [self addConstraints:[NSLayoutConstraint constraintsWithVisualFormat:@"H:|-kMargin-[_leftRangeLabel]-kMargin-[_slider]-kMargin-[_rightRangeLabel(==_leftRangeLabel)]-kMargin-|"
-                                                                     options:NSLayoutFormatAlignAllCenterY|NSLayoutFormatDirectionLeadingToTrailing
-                                                                     metrics:@{@"kMargin": @(kMargin)}
-                                                                       views:views]];
+        [self addConstraint:[NSLayoutConstraint constraintWithItem:self.rightRangeLabel
+                                                        attribute:NSLayoutAttributeCenterY
```

Comment:
Got it. And this is ok because the margin is 20 pt and we don't think the font will ever be bigger than 40pt line height, so won't extend beyond the bounds of the view.